_scripts/app1-8.R

```diff
@@ -0,0 +1,84 @@
+# Part A ----
+
+{
+  
+  rm(list=ls());
+  options(show.error.locations = TRUE);
+  
+  # Prompt user to input temperature
+  temp <- as.numeric(readline("What is the temperature? "))
+  
+  # check if the users value is greater than -20 but less than 30
+  if(temp > -20 & temp < 30)
+  {
+    cat("cold");
+  }
+  # check if the users value is equal to 30
+  else if(temp == 30)
+  {
+    cat("cold-ish");
+  }
+  # check if the users value is equal to 60
+  else if(temp == 60)
+  {
+    cat("hot-ish");
+  }
+  # check if the users value is greater than 60 but less than 100
+  else if(temp > 60 & temp <= 100)
+  {
+    cat("hot");
+  }
+  # check if the user has entered values outside the ranges listed above:
+  # E.g. < -20 & > 100
+  else 
+  {
+    cat("invalid value")
+  }
+
+  
+}
+
+# Part B ----
+
+{
+  
+  rm(list=ls());
+  options(show.error.locations = TRUE);
+  
+  # Prompt user to enter a letter grade:
+  # E.g. A-E.
+  letterGrade <- readline("What was the letter grade you achieved? ");
+  
+  # check if the user achieved a letter grade of 'A'
+  if(letterGrade == "A" | letterGrade == "a") # check for lower case entry
```

Comment:
Again, I would use || instead of | only because most other programming languages like C and Java) use || for this use case.  There are & and | operators in C and Java but they do something very different (they are called bitwise operators if you want to know more...)